src/Appointments.Console/Appointment Management System/Interfaces/IManagementRepository.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Appointment_Management_System.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Appointment_Management_System.Interfaces
+{
+    public interface IManagementRepository<T>
+    {
+        void Create(List<T> items, List<Customer>? item = null);
```

Comment:
The repository pattern dicates the underlying persistency layer is abstract to the caller.